pennylane/pauli/dla/center.py

```diff
@@ -54,25 +57,37 @@ def center(
     In this case, just ``X(0)``.
 
     >>> qml.center(g)
-    [X(0)]
+    [-0.9999999999999999 * X(0)]
 
     """
+    if len(g) < 2:
+        # A length-zero list has zero center, a length-one list has full center
+        return g
+
+    adjoint_repr = structure_constants(g, pauli)
+    # Compute kernel for adjoint rep of each DLA element
+    kernels = [null_space(ad_x) for ad_x in adjoint_repr]
+    # If any kernels vanishes, their overlap vanishes as well
+    if any(k.shape[1] == 0 for k in kernels):
+        return []
+    # Compute the complements of the kernels
+    supports = [null_space(k.T) for k in kernels]
+    # Combine all complements
+    combined_support = np.hstack(supports)
+    if combined_support.shape[1] == 0:
+        return g
+    # Compute the complement of the combined complements: It is the intersection of the kernels
+    center_coefficients = null_space(combined_support.T)
+
+    # Construct operators from numerical output and convert to desired format
+    res = [sum(c * x for c, x in zip(c_coeffs, g)) for c_coeffs in center_coefficients.T]
+
+    have_paulis = isinstance(g[0], (PauliWord, PauliSentence))
+    if pauli or have_paulis:
+        _ = [el.simplify() for el in res]
```

Comment:
So there are cases where there is 0-valued coefficients? (otherwise paulis are automatically simplified)

Reply:
Ah right 🙈